npc/quests/quests_gonryun.txt

```diff
@@ -3810,6 +3812,7 @@ OnTouch_:
 		mes "- You have found an old knife -";
 		close2;
 		getitem 1201,1; //Knife
+		changequest 10034,10035;
```

Comment:
Pleae move `changequest` near `set nakha,2;` (few lines above) to avoid a potential warning

Reply:
I moved it up, but I'm kind of wondering if it wouldn't have made more sense for me to move `set nakha,2;` down instead. It seems like the more natural flow of things, to first read the window that tells you that you find something, then close that window, get the item and the new questlog entry.